bpf/kmesh/workload/include/workload.h

```diff
@@ -24,11 +24,12 @@
 
 #define MAX_PORT_COUNT   10
 
+// Specifically, for Pod to Pod access: frontend_key:{ipv4:<PodIP>, service_port:0}, frontend_value:{service_id:0}
 // frontend map
 typedef struct
 {
-    __u32 ipv4;			// service vip
-    __u32 service_port;	// service port
+    __u32 ipv4;         // service ip, specifically: PodIP
+    __u32 service_port;	// service port, specifically: 0
```

Comment:
Sorry i cannot understand what does 0 mean

For a workload, we still need to match the port,

Reply:
The workload message of a Pod that not belong to any service does not contain port info, so when stored, set the port as 0.